api/controllers/config_controller.go

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/numary/ledger/api/schemas"
+	"github.com/numary/ledger/api/services"
+)
+
+// ConfigController -
+type ConfigController struct {
+	BaseController
+	configService *services.ConfigService
+}
+
+// NewConfigController -
+func NewConfigController(
+	configService *services.ConfigService,
+) *ConfigController {
+	return &ConfigController{
+		configService: configService,
+	}
+}
+
+// CreateConfigController -
+func CreateConfigController() *ConfigController {
+	return NewConfigController(
+		services.CreateConfigService(),
```

Comment:
I think I'd be inclined to have services created and injected via fx. What do you think?

Reply:
The services layer from API package is now deleted, this extra layer is not that useful yet.